packages/camera/camera_avfoundation/ios/Classes/FLTCam.h

```diff
@@ -14,6 +14,35 @@
 
 NS_ASSUME_NONNULL_BEGIN
 
+/**
+ * A class that manages media settings state and performs media settings operations.
+ */
+@interface FLTCamMediaSettingsProvider : NSObject
```

Comment:
Please make this new class its own pair of files.

Reply:
done with own pair of files.

Reply:
even more - splitted into settings and wrapper parts.